### PR TITLE
[FEATURE] Demand maxPricePerMwh

### PR DESCRIPTION
**Pull request by josipbagaric**

### CODE REVIEW

File: src/components/CertificateTable.tsx [hunk 1]
```diff
@@ -118,21 +118,21 @@ export class CertificateTable extends PaginatedLoader<ICertificateTableProps, IC
...
-
         if (this.props.selectedState === SelectedState.ForDemand && this.props.demand) {
             await this.initMatchingCertificates(this.props.demand);
         }
+
+        await super.componentDidMount();
     }
 
-    async componentDidUpdate(newProps: ICertificateTableProps) {
+    async componentDidUpdate(newProps: ICertificateTableProps, prevState: ICertificatesState) {
```
Comment (e00dan) [inline src/components/CertificateTable.tsx]: Unnecessary declaration of prevState

File: src/elements/Table/Table.tsx [hunk 2]
```diff
@@ -487,6 +487,13 @@ export class Table extends React.Component<IProps, State> {
                                                             />
                                                         </div>
                                                     )}
+                                                {item.input.type === 'number' && (
+                                                    <div>
+                                                        <input type="number"
```
Comment (e00dan) [inline src/elements/Table/Table.tsx:493]: From my experience we should avoid `type="number"`, there's a lot of problems with it especially when it comes to comma vs dot in floats

**Review by e00dan — APPROVED** [review]

File: src/elements/Table/Table.tsx (hunk 2 above)
Comment (josipbagaric) [reply src/elements/Table/Table.tsx:493]: I agree, put it here for validation simplicity - should be changed/addressed after the MVP is over.

Comment (josipbagaric) [reply src/elements/Table/Table.tsx:493]: Added a note `// TO-DO: Deprecate the use of input type number after POC`

File: src/components/CertificateTable.tsx (hunk 1 above)
Comment (josipbagaric) [reply src/components/CertificateTable.tsx]: Thanks, forgot to remove that after testing.

**Pull request merged**